src/lomc.jl

```diff
@@ -18,8 +18,10 @@ See also [`QMCState`](@ref), [`ReplicaStrategy`](@ref), [`replica_stats`](@ref),
 [`lomc!`](@ref).
 """
 mutable struct ReplicaState{H,T,V,W,R<:FciqmcRunStrategy{T}}
+    # Future TODO: rename these fields, add interface for accessing them.
     hamiltonian::H
     v::V       # vector
+    pv::V      # previous vector.
     w::W       # working memory. Maybe working memories could be shared among replicas?
```

Comment:
```suggestion
    wm::W       # working memory. Maybe working memories could be shared among replicas?
```
I would suggest renaming to `wm`  as  this is what the keyword argument  in `lomc!` is called. It also makes the function more  apparent.